gnmi/gnmi.go

```diff
@@ -338,3 +326,24 @@ func (s *Server) Get(ctx context.Context, req *gpb.GetRequest) (*gpb.GetResponse
 		return nil, status.Errorf(codes.DataLoss, "Unknown message type: %T", resp)
 	}
 }
+
+// StartReconcilers starts all the reconcilers.
+func (s *Server) StartReconcilers(ctx context.Context) error {
+	c := gnmiclient.New(s)
```

Comment:
Should this be an input to avoid importing the client package?

Reply:
moved to gnmi package